### DISCUSSION TITLE
Webpack perfomance recommendations

### DISCUSSION THREAD

**Discussion by E0565304** [SAP/luigi#3214] [Q&A]
Hi, can anyone help me how to solve that ??
![image](https://user-images.githubusercontent.com/126490465/224885886-4a62022a-7004-49ac-a2d2-5cfdc368bcbb.png)

in my webback.config.js i do have changes :   performance: {
    maxAssetSize: 1500000,
    maxEntrypointSize: 600000,
  },

i have this error after switching from luigi version 1 to version 2.

**Comment by hardl** [comment]
Hi, do you have a custom assetFilter in your webpack config? 
Because .map files should be ignored by default (see https://webpack.js.org/configuration/performance/#performanceassetfilter).
